COMMUNITY_BOARD_SEAT_EXPECTATIONS.md

```diff
@@ -0,0 +1,56 @@
+# Expectations of the Community Board of Directors representatives
+
+The OpenJS Foundation bylaws provide for up to three board seats to be filled by the community. This is a brief description of why they exist, and what responsibilities they carry.
+
+## About the OpenJS Foundation Board of Directors
+
+As a non-profit, the OpenJS Foundation is overseen by a Board of Directors. Their responsibilities range from managing the business oversight of the Foundation, including budget, marketing, and Foundation staff, to setting legal policy. The Board has a responsibility to ensure that Foundation activities support its purpose as defined in the bylaws:
+
+*   Promote the widespread adoption and continued development of key JavaScript solutions and related technologies worldwide.
+*   Facilitate collaboration within the JavaScript and web development community.
+*   Create a center of gravity for open source projects throughout the end-to-end JavaScript ecosystem guiding them toward open governance and diverse collaborator bases.
+*   Host the infrastructure to support hosted JavaScript open source projects (the “Projects”).
+*   Enable, through advancement of Projects and strategic partnerships, an open and accessible web.
+*   Undertake such other activities as may from time to time be appropriate to further the purposes and achieve the goals set forth above.
+
+## Representations on the Board of Directors
+
+There are three ways by which one becomes a Director.
+
+*   Your company is a Platinum member, and you are appointed.
+*   Your company is a Gold or Silver member, and you are elected by the other Gold or Silver members.
+*   You are chosen by the community to fill one of the community seats.
+
+## Responsibilities of Directors
+
+All Directors have a responsibility to represent the organization or community group which appointed or voted them into their role. In addition, all Directors have a responsibility to represent the best interests of the organization and our development communities.
+
+At a high level, a Director’s responsibilities include:
+
+*   Attending as many board meetings as possible. The board generally meets once per month for two hours. Board meetings require quorum in order to vote, and every meeting includes a vote on the prior meeting’s minutes.
+*   Be in touch with your constituents. The most effective board members are in tune with the facts of their community, and opinions held by the people they represent. Their objective is to represent these faithfully and articulately in board matters. Community Directors are expected to be active participants in CPC meetings.
+*   Be in touch with your legal counsel. The OpenJS Foundation has legal counsel to advise the Foundation itself. However, for matters of policy, corporate Directors should discuss their positions with corporate counsel to ensure they are aligned with their organization’s positions.
```

Comment:
Maybe clarify what this means for board members who are representing the CPC or the community. Are they supposed to be aligned with their organization or the interest of their constituency? If the latter, where can they get legal advice from?